src/components/implementation/tests/unit_cxx/Makefile

```diff
@@ -0,0 +1,18 @@
+C_OBJS=cxx.o
+COMPONENT=cxx_test.o
+INTERFACES=
+DEPENDENCIES=cbuf_mgr printc sched valloc mem_mgr_large torrent evt timed_blk lock
+IF_LIB=./libstdc++.a ./libsupc++.a 
+OBJLIBS += $(POSIX_LIB)
+ADDITIONAL_LIBS=-lcobj_format -lcos_kernel_api
+
+pre: first all
+first:
+	$(info |     [CP]   copying libsupc++.a from libcxx)
+	cp $(LIBSUPCXXOBJ) .
+	$(info |     [CP]   copying libstdc++.a from libcxx)
+	cp $(LIBSTDCXXOBJ) .
```

Comment:
Is this really the best way to do this? Shouldn't this leverage the existing library system and be in ADDITIONAL_LIBS?